src/pages/office/cat-be/office.cat-be.page.ts

```diff
@@ -0,0 +1,661 @@
+import { behaviourMap } from '../office-behaviour-map';
+import { ActivityCodeModel, activityCodeModelList } from '../components/activity-code/activity-code.constants';
+import { CompetencyOutcome } from '../../../shared/models/competency-outcome';
+import { startsWith } from 'lodash';
+import { getRekeyIndicator } from '../../../modules/tests/rekey/rekey.reducer';
+import { isRekey } from '../../../modules/tests/rekey/rekey.selector';
+import { CAT_BE, JOURNAL_PAGE } from '../../page-names.constants';
+import { SetActivityCode } from '../../../modules/tests/activity-code/activity-code.actions';
+import { VehicleChecksQuestion } from '../../../providers/question/vehicle-checks-question.model';
+import { TestCategory } from '../../../shared/models/test-category';
+import { FaultCountProvider } from '../../../providers/fault-count/fault-count';
+import { getTestCategory } from '../../../modules/tests/category/category.reducer';
+
+interface OfficePageState {
+  activityCode$: Observable<ActivityCodeModel>;
+  startTime$: Observable<string>;
+  testOutcome$: Observable<string>;
+  testOutcomeText$: Observable<string>;
+  isPassed$: Observable<boolean>;
+  isTestOutcomeSet$: Observable<boolean>;
+  candidateName$: Observable<string>;
+  candidateDriverNumber$: Observable<string>;
+  routeNumber$: Observable<number>;
+  displayRouteNumber$: Observable<boolean>;
+  displayIndependentDriving$: Observable<boolean>;
+  displayCandidateDescription$: Observable<boolean>;
+  displayIdentification$: Observable<boolean>;
+  displayShowMeQuestion$: Observable<boolean>;
+  displayTellMeQuestion$: Observable<boolean>;
+  displayWeatherConditions$: Observable<boolean>;
+  displayAdditionalInformation$: Observable<boolean>;
+  displayEco$: Observable<boolean>;
+  displayEta$: Observable<boolean>;
+  displayDrivingFault$: Observable<boolean>;
+  displaySeriousFault$: Observable<boolean>;
+  displayDangerousFault$: Observable<boolean>;
+  identification$: Observable<Identification>;
+  independentDriving$: Observable<IndependentDriving>;
+  candidateDescription$: Observable<string>;
+  additionalInformation$: Observable<string>;
+  showMeQuestion$: Observable<VehicleChecksQuestion>;
+  showMeQuestionOptions$: Observable<VehicleChecksQuestion[]>;
+  tellMeQuestionText$: Observable<string>;
+  etaFaults$: Observable<string>;
+  ecoFaults$: Observable<string>;
+  drivingFaults$: Observable<MultiFaultAssignableCompetency[]>;
+  drivingFaultCount$: Observable<number>;
+  displayDrivingFaultComments$: Observable<boolean>;
+  weatherConditions$: Observable<WeatherConditions[]>;
+  dangerousFaults$: Observable<(MultiFaultAssignableCompetency | CommentedCompetency)[]>;
+  seriousFaults$: Observable<(MultiFaultAssignableCompetency | CommentedCompetency)[]>;
+  isRekey$: Observable<boolean>;
+}
+
+@IonicPage()
+@Component({
+  selector: 'page-office',
+  templateUrl: 'office.cat-be.page.html',
+})
+export class OfficeCatBEPage extends PracticeableBasePageComponent {
```

Comment:
Cat BE pages should be BasePageComponent not PracticeableBasePageComponent. Just needs a couple of things removed, including practice header in the html.